app/lib/account/consent_backend.dart

```diff
@@ -38,8 +38,8 @@ class ConsentBackend {
   ConsentBackend(this._db);
 
   /// Returns the consent details.
-  Future<api.Consent> getConsent(String consentId) async {
-    final user = await requireAuthenticatedUser();
+  Future<api.Consent> getConsent(String consentId, {User user}) async {
```

Comment:
I'd rather we duplicate.. or that we have `getConsentApi` which calls `requireAuthenticatedUser` and then calls `getConsent(user)`.

Notably, I don't like the implication that you can avoid authorization by passing a user object.